src/tests/functional/plugin/cpu/single_layer_tests/lstm_cell.cpp

```diff
@@ -42,91 +66,112 @@ class LSTMCellLayerCPUTest : public testing::WithParamInterface<LSTMCellCpuSpeci
 
 protected:
     void SetUp() override {
-        LayerTestsDefinitions::LSTMCellParams basicParamsSet;
+        std::vector<InputShape> inputShapes;
+        bool decompose;
+        std::vector<std::string> activations;
+        float clip = 0.f;
+        ElementType netPrecision;
         CPUSpecificParams cpuParams;
         std::map<std::string, std::string> additionalConfig;
+        abs_threshold = 0.05;
 
-        bool should_decompose;
-        size_t batch;
-        size_t hidden_size;
-        size_t input_size;
-        std::vector<std::string> activations;
-        std::vector<float> activations_alpha;
-        std::vector<float> activations_beta;
-        float clip;
-        InferenceEngine::Precision netPrecision;
-        threshold = 0.05;
-
-        std::tie(basicParamsSet, cpuParams, additionalConfig) = this->GetParam();
+        std::tie(inputShapes, decompose, activations, clip, netPrecision, cpuParams, additionalConfig) = this->GetParam();
         std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
-        std::tie(should_decompose, batch, hidden_size, input_size, activations, clip, netPrecision, targetDevice) = basicParamsSet;
+        targetDevice = CommonTestUtils::DEVICE_CPU;
 
-        std::vector<std::vector<size_t>> inputShapes = {
-            {{batch, input_size}, {batch, hidden_size}, {batch, hidden_size}, {4 * hidden_size, input_size}, {4 * hidden_size, hidden_size}, {4 * hidden_size}},
-        };
+        init_input_shapes(inputShapes);
 
         configuration.insert(additionalConfig.begin(), additionalConfig.end());
 
-        if (additionalConfig[PluginConfigParams::KEY_ENFORCE_BF16] == PluginConfigParams::YES) {
-            inPrc = outPrc = Precision::BF16;
-        } else {
-            inPrc = outPrc = netPrecision;
-        }
-
-        selectedType += "_";
-        selectedType += outPrc.name();
-
-        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(Precision::FP32);
-        auto params = ngraph::builder::makeParams(ngPrc, {inputShapes[0], inputShapes[1], inputShapes[2]});
-        std::vector<ngraph::Shape> WRB = {inputShapes[3], inputShapes[4], inputShapes[5]};
+        size_t hiddenSize = inputDynamicShapes[1][1].get_length();
+        size_t inputSize = inputDynamicShapes.front()[1].get_length();
 
-        auto lstm_cell = ngraph::builder::makeLSTM(
-            ngraph::helpers::convert2OutputVector(ngraph::helpers::castOps2Nodes(params)), WRB, hidden_size, activations, {}, {}, clip);
+//        if (additionalConfig[InferenceEngine::PluginConfigParams::KEY_ENFORCE_BF16] == InferenceEngine::PluginConfigParams::YES) {
```

Comment:
uncomment please

Reply:
That does not work properly with new tests API.